packages/manager-components/src/components/layout/dashboard/dashboard.spec.tsx

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable no-return-await */
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
```

Comment:
by updating jest config / ts config you should be able to avoid having to import React globally

Reply:
Sure, It has been correct in tsconfig and jest config 👍